lang/LangPrimSource/SC_Ableton_Link.cpp

```diff
@@ -0,0 +1,195 @@
+#include "PyrKernel.h"
+#include "PyrSched.h"
+#include "GC.h"
+#include "PyrPrimitive.h"
+#include "PyrSymbol.h"
+
+#include "SCBase.h"
+#include "SC_Clock.h"
+
+#ifdef SC_ABLETON_LINK
+
+#include <ableton/Link.hpp>
+
+PyrSymbol* s_LinkTempoChanged;
```

Comment:
it's clearest to name this `s_prTempoChanged`, since that's the symbol it will refer to.

Reply:
actually, I think this is unnecessary optimization; this is only needed during clock construction which is a relatively uncommon task. we can just put a `getsym` directly in the constructor

Reply:
It's used in the anonymous function defined as an argument of `mLink.setTempoCallback`. So it's rather needed when the session tempo is changed.

Reply:
ok, I still think that is a rare enough occurrence that it doesn't warrant this optimization. it makes the file more difficult to read because the information is distributed at the top and bottom of the file and also at the usage site